src/main/webapp/WEB-INF/tags/instructor/results/filterEditButton.tag

```diff
@@ -0,0 +1,21 @@
+<%@ tag description="instructorFeedbackResultsTop - Filter Panel Edit Button" %>
```

Comment:
It'd be easier to revert this change first (i.e. it's okay if the expand/collapse button stays inlined within the filter panel for now) for a cleaner merge with #7665.